src/com/hotmail/kalebmarc/textfighter/item/FirstAid.java

```diff
@@ -6,7 +6,7 @@
 import com.hotmail.kalebmarc.textfighter.player.Stats;
 import com.hotmail.kalebmarc.textfighter.player.Xp;
 
-public class FirstAid {
+public class    FirstAid {

```

Comment:
i dont think you mean to leave these spaces here
```suggestion
public class FirstAid {

```